assets/css/styles.css

```diff
@@ -0,0 +1,4 @@
+@import '../../components/page/page.css';
+@import '../../components/header/header.css';
+@import '../../components/context/context.css';
+@import '../../components/footer/footer.css';
```

Comment:
Is this common practice to include all blocks css in one stylesheet?

Reply:
yes, cos' usually we use some bundler which can split styles later into separate chunks - 1 per page / component, if it's really necessary.